### PR TITLE
Solve issue #1

### PR DESCRIPTION
**Pull request by phpguru**
You were correct. It works fine from the current Lambda AMI. I've updated the create_php_binary with my experience.

### CODE REVIEW

**Comment by akrabat** [comment]
Rebased due to conflict

File: doc/create_php_binary.md [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
-2. Copy compile_php.sh to instance
+1. Create a EC2 large instance based on 
+[the current supported version AMI](https://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html).
+2. Connect to the instance after it boots to accept it to your known hosts:
 
-    From this project's root directory:
-    
         $ export AWS_IP=ec2-user@{ipaddress}
         $ export SSH_KEY_FILE=~/.ssh/aws-key.rsa
+        $ ssh -i $SSH_KEY_FILE $AWS_IP
```
Comment (akrabat) [inline doc/create_php_binary.md:9]: Why `ssh` into the box?

Comment (phpguru) [reply doc/create_php_binary.md:9]: The scp doesn’t work until you add the host to known_hosts, for me anyway. It just hangs. Once I ssh’d in and accepted the entry the scp command worked fine.

Comment (akrabat) [reply doc/create_php_binary.md:9]: ah

**Pull request merged**